src/core_modules/capture-core/metaDataStoreLoaders/baseLoader/metaDataLoader.js

```diff
@@ -1,74 +1,47 @@
 // @flow
```

Comment:
Can we make a distinction between load and store here? 

For me store would be that it wont return any data and load would be that it returns some data. Now its mixed up for me. 

Can we please think of it?

Reply:
I've been thinking of it this way: In the process of loading parts of the app, you're storing to the cache. I don't see why a load function would need to return some data.